internal/schemamd/testdata/framework_types.md

```diff
@@ -0,0 +1,126 @@
+
+<a id="nestedblock--list_nested_block"></a>
+### Nested Schema for `list_nested_block`
+
+Optional:
+
+- `list_nested_block_attribute` (String) example list nested block attribute
+- `list_nested_block_attribute_with_default` (String) example list nested block attribute with default
+- `nested_list_block` (Block List) (see [below for nested schema](#nestedblock--list_nested_block--nested_list_block))
+
+<a id="nestedblock--list_nested_block--nested_list_block"></a>
+### Nested Schema for `list_nested_block.nested_list_block`
+
+Optional:
+
+- `nested_block_string_attribute` (String) example nested block string attribute
+
+
+
+<a id="nestedblock--list_nested_block_sensitive_nested_attribute"></a>
+### Nested Schema for `list_nested_block_sensitive_nested_attribute`
+
+Optional:
+
+- `list_nested_block_attribute` (String) example list nested block attribute
+- `list_nested_block_sensitive_attribute` (String, Sensitive) example sensitive list nested block attribute
+
+
+<a id="nestedatt--object_attribute"></a>
+### Nested Schema for `object_attribute`
+
+Optional:
+
+- `object_attribute_attribute` (String)
+
+
+<a id="nestedatt--object_attribute_with_nested_object_attribute"></a>
+### Nested Schema for `object_attribute_with_nested_object_attribute`
+
+Optional:
+
+- `nested_object` (Object) (see [below for nested schema](#nestedobjatt--object_attribute_with_nested_object_attribute--nested_object))
+- `object_attribute_attribute` (String)
+
+<a id="nestedobjatt--object_attribute_with_nested_object_attribute--nested_object"></a>
+### Nested Schema for `object_attribute_with_nested_object_attribute.nested_object`
+
+Optional:
+
+- `nested_object_attribute` (String)
+
+
+
+<a id="nestedatt--sensitive_object_attribute"></a>
+### Nested Schema for `sensitive_object_attribute`
+
+Optional:
+
+- `object_attribute_attribute` (String)
+
+
+<a id="nestedblock--set_nested_block"></a>
+### Nested Schema for `set_nested_block`
+
+Optional:
+
+- `set_nested_block_attribute` (String) example set nested block attribute
+
+
+<a id="nestedblock--single_nested_block"></a>
+### Nested Schema for `single_nested_block`
+
+Optional:
+
+- `single_nested_block_attribute` (String) example single nested block attribute
+
+
+<a id="nestedblock--single_nested_block_sensitive_nested_attribute"></a>
+### Nested Schema for `single_nested_block_sensitive_nested_attribute`
+
+Optional:
+
+- `single_nested_block_attribute` (String) example single nested block attribute
+- `single_nested_block_sensitive_attribute` (String, Sensitive) example sensitive single nested block attribute
+
+
+<a id="nestedblock--set_nested_block_sensitive_nested_attribute"></a>
+### Nested Schema for `set_nested_block_sensitive_nested_attribute`
+
+Read-Only:
+
+- `set_nested_block_attribute` (String) example set nested block attribute
+- `set_nested_block_sensitive_attribute` (String, Sensitive) example sensitive set nested block attribute
```

Comment:
Non-blocking: I do find it curious how these become un-alphabetically ordered. 🤔  Certainly not for this PR, but it might be nice to see if we can fix that at some point for practitioners who might scroll instead of clicking the link. 👍